src/components/NcAppNavigation/NcAppNavigation.vue

```diff
@@ -76,6 +73,9 @@ emit('toggle-navigation', {
 			<!-- Footer for e.g. AppNavigationSettings -->
 			<slot name="footer" />
 		</nav>
+		<div class="app-navigation__toggle-wrapper">
+			<NcAppNavigationToggle :open="open" @update:open="toggleNavigation" />
+		</div>
```

Comment:
This `div` wrapper can be removed (forgot to do it in https://github.com/nextcloud-libraries/nextcloud-vue/pull/4779)

Reply:
Positioned and works the same without div, updated